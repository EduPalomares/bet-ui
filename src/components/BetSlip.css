.betslip {
	width: 100%;
	height: 100%;
	max-height: 100%;
	position: absolute;
	top: 0px;
}

.betslip-panel {
	width: 100%;
	height: 100%;
	position: absolute;
	top: 0;
	right: 0;
	overflow:hidden;
	z-index:1;
}

.betslip.collapsed {
	max-height:0vh;
	transition:all .0s .5s;
}

.betslip.collapsed .betslip-icon {
	opacity:1;
	transition: all .7s;
	z-index:2;
}

.betslip-panel .bp-bg {
	width: 100%;
	height: 100%;
	position:absolute;
	background:rgba(0,0,0,.3);
	opacity:1;
	transition: all .5s; 
}

.betslip.collapsed .betslip-panel .bp-bg {
	opacity:0;
	transition: all .2s; 
}

.betslip-panel .bp-inner {
	width: 85%;
	height: 100%;
	position: absolute;
	top: 0;
	right: 0;
	padding-top: 60px;
	background:#FFF;
	border-left:4px solid #79879c;
	transition: right .5s; 
}

.betslip.collapsed .betslip-panel .bp-inner {
	right: -100%;
	transition: right .3s; 
}

.betslip-panel .close  {
	width: 25px;
	height: 25px;
	position: absolute;
	top: 25px;
	right: 25px;
	z-index:999;
	overflow: hidden;
	text-indent: 1em;
	font-size: 0.75em;
	border: none;
	background: transparent;
	color: transparent;
	cursor: pointer;
}

.betslip-panel .close::before, .betslip-panel .close::after  {
	content: '';
	width: 4px;
	height: 100%;
	position: absolute;
	top: 0;
	left: 50%;
	background: #5C5C5C;
}

.betslip-panel .close::before {
	transform: rotate(45deg);
}

.betslip-panel .close::after {
	transform: rotate(-45deg);
}