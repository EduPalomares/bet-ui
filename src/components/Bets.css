.bets {
	width: 443px;
	height: 902px;
	position: absolute;
	top: 50%;
	left: 50%;
	transform: translate(-50%,-50%);
	background: url(img/bets-bg.jpg) center/cover no-repeat;
	font-family: Helvetica, Arial, 'Verdana'; 
}

.bets .inner {
	width: 375px;
	height: 667px;
	position: absolute;
	top: 115px;
	left: 33px;
	padding-top: 70px;
	background: #FFF;
	overflow: hidden;
}