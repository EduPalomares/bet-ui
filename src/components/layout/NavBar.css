.navbar {
	width: 375px;
    height: 56px;
	position: absolute;
	top:0;
	border-bottom: 4px solid #5C5C5C;
}

.navbar-icon {
	position: absolute;
	right:15px;
	padding-top: 13px;
	cursor: pointer;
	z-index:1;
}

.bar1, .bar2, .bar3 {
    width: 35px;
    height: 5px;
    background-color: #5C5C5C;
    margin-top: 0px;
	margin-bottom: 6px;
	transition: 0.4s;
}

.change .bar1 {
    -webkit-transform: rotate(-45deg) translate(-9px, 6px);
    transform: rotate(-45deg) translate(-9px, 6px);
}

.change .bar2  {
	opacity: 0;
}

.change .bar3 {
    -webkit-transform: rotate(45deg) translate(-8px, -8px);
    transform: rotate(45deg) translate(-8px, -8px);
}