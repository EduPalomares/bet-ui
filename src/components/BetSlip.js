import React, { Component } from 'react';
import { Consumer } from './Provider';
import Bet from './Bet';
import './BetSlip.css';

export default class BetSlip extends Component {
	render() {
		return (
			<Consumer>
				{
					context => {
					
						const userBets = context.betsIds.map(function(betId, index) {
								const bet = context.possibleBets[betId];
								const betMarketNameShort = (bet.marketName==='Team to Win') ? ' to Win' : ' to Score';
								return <Bet key={index} id={betId} betName={bet.selectionName+betMarketNameShort} betPrice={bet.selectionPrice} />
							}
						);
		
						return (
							<div className={(!context.panelIsOpen) ? "betslip collapsed" : "betslip"} >
								<div className="betslip-panel">
									<div className="bp-bg"></div>
									<div className="bp-inner">
										<div className="close" onClick={context.closePanel}></div>
										{userBets}
									</div>
								</div>
							</div>
						);
					}	
				}
			</Consumer>
		);
	}
}