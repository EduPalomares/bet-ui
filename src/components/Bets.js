import React, { Component } from 'react';
import { Provider } from './Provider';
import NavBar from './layout/NavBar';
import Event from './Event';
import BetSlip from './BetSlip';

import data from './assets/data.json';
import './Bets.css';


export default class Bets extends Component {

	render() {
		
		const events = data.map((event) =>
			(event.markets.length>0) ? <Event key={event.id} event={event} /> : ''
		);

		return (
			<Provider>
				<div className="bets">
					<div className="inner">
						<NavBar />
						<BetSlip />
						{events}
					</div>
				</div>
			</Provider>
		);
	}
}