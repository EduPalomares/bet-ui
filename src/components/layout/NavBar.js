import React, { Component } from 'react';
import { Consumer } from '../Provider';
import './NavBar.css';

export default class NavBar extends Component {
	render() {
		return (
			<Consumer>
				{ 
					context => 
						<div className="navbar">
							<div className="navbar-icon" onClick={context.openPanel}>
								<div className="bar1"></div>
								<div className="bar2"></div>
								<div className="bar3"></div>
							</div>
						</div>
				}
			</Consumer>
		);
	}
}