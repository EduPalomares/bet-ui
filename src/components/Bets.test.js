import React from 'react';
import Bets from './Bets';
import NavBar from './layout/NavBar';
import BetSlip from './BetSlip';

const wrapper = shallow(<Bets />);

test('render match snapshot', () => {
	expect(wrapper).toMatchSnapshot();
});

test('render structure right', () => {
	expect(wrapper.contains(<NavBar />)).toEqual(true);
	expect(wrapper.contains(<BetSlip />)).toEqual(true);
});
