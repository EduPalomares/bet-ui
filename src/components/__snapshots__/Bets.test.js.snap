// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render match snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Bets />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "class",
    "props": Object {
      "children": <div
        className="bets"
>
        <div
                className="inner"
        >
                <NavBar />
                <BetSlip />
                <Event
                        event={
                                Object {
                                        "id": "EVT_1",
                                        "markets": Array [
                                          Object {
                                            "id": "MKT_1",
                                            "name": "Team to Win",
                                            "selections": Array [
                                              Object {
                                                "id": "SEL_1",
                                                "name": "Real Madrid",
                                                "price": 1.23,
                                              },
                                              Object {
                                                "id": "SEL_2",
                                                "name": "Barcelona",
                                                "price": 2.05,
                                              },
                                            ],
                                          },
                                          Object {
                                            "id": "MKT_2",
                                            "name": "Player to Score First",
                                            "selections": Array [
                                              Object {
                                                "id": "SEL_3",
                                                "name": "Ronaldo",
                                                "price": 1.15,
                                              },
                                              Object {
                                                "id": "SEL_4",
                                                "name": "Messi",
                                                "price": 1.3,
                                              },
                                              Object {
                                                "id": "SEL_5",
                                                "name": "Bale",
                                                "price": 1.35,
                                              },
                                            ],
                                          },
                                        ],
                                        "name": "Real Madrid vs Barcelona",
                                      }
                        }
                />
                <Event
                        event={
                                Object {
                                        "id": "EVT_2",
                                        "markets": Array [
                                          Object {
                                            "id": "MKT_3",
                                            "name": "Team to Win",
                                            "selections": Array [
                                              Object {
                                                "id": "SEL_6",
                                                "name": "Atletico",
                                                "price": 1.4,
                                              },
                                              Object {
                                                "id": "SEL_7",
                                                "name": "Malaga",
                                                "price": 3.05,
                                              },
                                            ],
                                          },
                                        ],
                                        "name": "Atletico Madrid vs Malaga",
                                      }
                        }
                />
                
        </div>
</div>,
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <div
          className="inner"
>
          <NavBar />
          <BetSlip />
          <Event
                    event={
                              Object {
                                        "id": "EVT_1",
                                        "markets": Array [
                                          Object {
                                            "id": "MKT_1",
                                            "name": "Team to Win",
                                            "selections": Array [
                                              Object {
                                                "id": "SEL_1",
                                                "name": "Real Madrid",
                                                "price": 1.23,
                                              },
                                              Object {
                                                "id": "SEL_2",
                                                "name": "Barcelona",
                                                "price": 2.05,
                                              },
                                            ],
                                          },
                                          Object {
                                            "id": "MKT_2",
                                            "name": "Player to Score First",
                                            "selections": Array [
                                              Object {
                                                "id": "SEL_3",
                                                "name": "Ronaldo",
                                                "price": 1.15,
                                              },
                                              Object {
                                                "id": "SEL_4",
                                                "name": "Messi",
                                                "price": 1.3,
                                              },
                                              Object {
                                                "id": "SEL_5",
                                                "name": "Bale",
                                                "price": 1.35,
                                              },
                                            ],
                                          },
                                        ],
                                        "name": "Real Madrid vs Barcelona",
                                      }
                    }
          />
          <Event
                    event={
                              Object {
                                        "id": "EVT_2",
                                        "markets": Array [
                                          Object {
                                            "id": "MKT_3",
                                            "name": "Team to Win",
                                            "selections": Array [
                                              Object {
                                                "id": "SEL_6",
                                                "name": "Atletico",
                                                "price": 1.4,
                                              },
                                              Object {
                                                "id": "SEL_7",
                                                "name": "Malaga",
                                                "price": 3.05,
                                              },
                                            ],
                                          },
                                        ],
                                        "name": "Atletico Madrid vs Malaga",
                                      }
                    }
          />
          
</div>,
        "className": "bets",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": Array [
            <NavBar />,
            <BetSlip />,
            Array [
              <Event
                event={
                                Object {
                                                "id": "EVT_1",
                                                "markets": Array [
                                                  Object {
                                                    "id": "MKT_1",
                                                    "name": "Team to Win",
                                                    "selections": Array [
                                                      Object {
                                                        "id": "SEL_1",
                                                        "name": "Real Madrid",
                                                        "price": 1.23,
                                                      },
                                                      Object {
                                                        "id": "SEL_2",
                                                        "name": "Barcelona",
                                                        "price": 2.05,
                                                      },
                                                    ],
                                                  },
                                                  Object {
                                                    "id": "MKT_2",
                                                    "name": "Player to Score First",
                                                    "selections": Array [
                                                      Object {
                                                        "id": "SEL_3",
                                                        "name": "Ronaldo",
                                                        "price": 1.15,
                                                      },
                                                      Object {
                                                        "id": "SEL_4",
                                                        "name": "Messi",
                                                        "price": 1.3,
                                                      },
                                                      Object {
                                                        "id": "SEL_5",
                                                        "name": "Bale",
                                                        "price": 1.35,
                                                      },
                                                    ],
                                                  },
                                                ],
                                                "name": "Real Madrid vs Barcelona",
                                              }
                }
/>,
              <Event
                event={
                                Object {
                                                "id": "EVT_2",
                                                "markets": Array [
                                                  Object {
                                                    "id": "MKT_3",
                                                    "name": "Team to Win",
                                                    "selections": Array [
                                                      Object {
                                                        "id": "SEL_6",
                                                        "name": "Atletico",
                                                        "price": 1.4,
                                                      },
                                                      Object {
                                                        "id": "SEL_7",
                                                        "name": "Malaga",
                                                        "price": 3.05,
                                                      },
                                                    ],
                                                  },
                                                ],
                                                "name": "Atletico Madrid vs Malaga",
                                              }
                }
/>,
              "",
            ],
          ],
          "className": "inner",
        },
        "ref": null,
        "rendered": Array [
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": undefined,
            "nodeType": "class",
            "props": Object {},
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "EVT_1",
            "nodeType": "class",
            "props": Object {
              "event": Object {
                "id": "EVT_1",
                "markets": Array [
                  Object {
                    "id": "MKT_1",
                    "name": "Team to Win",
                    "selections": Array [
                      Object {
                        "id": "SEL_1",
                        "name": "Real Madrid",
                        "price": 1.23,
                      },
                      Object {
                        "id": "SEL_2",
                        "name": "Barcelona",
                        "price": 2.05,
                      },
                    ],
                  },
                  Object {
                    "id": "MKT_2",
                    "name": "Player to Score First",
                    "selections": Array [
                      Object {
                        "id": "SEL_3",
                        "name": "Ronaldo",
                        "price": 1.15,
                      },
                      Object {
                        "id": "SEL_4",
                        "name": "Messi",
                        "price": 1.3,
                      },
                      Object {
                        "id": "SEL_5",
                        "name": "Bale",
                        "price": 1.35,
                      },
                    ],
                  },
                ],
                "name": "Real Madrid vs Barcelona",
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          Object {
            "instance": null,
            "key": "EVT_2",
            "nodeType": "class",
            "props": Object {
              "event": Object {
                "id": "EVT_2",
                "markets": Array [
                  Object {
                    "id": "MKT_3",
                    "name": "Team to Win",
                    "selections": Array [
                      Object {
                        "id": "SEL_6",
                        "name": "Atletico",
                        "price": 1.4,
                      },
                      Object {
                        "id": "SEL_7",
                        "name": "Malaga",
                        "price": 3.05,
                      },
                    ],
                  },
                ],
                "name": "Atletico Madrid vs Malaga",
              },
            },
            "ref": null,
            "rendered": null,
            "type": [Function],
          },
          "",
        ],
        "type": "div",
      },
      "type": "div",
    },
    "type": [Function],
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "class",
      "props": Object {
        "children": <div
          className="bets"
>
          <div
                    className="inner"
          >
                    <NavBar />
                    <BetSlip />
                    <Event
                              event={
                                        Object {
                                                  "id": "EVT_1",
                                                  "markets": Array [
                                                    Object {
                                                      "id": "MKT_1",
                                                      "name": "Team to Win",
                                                      "selections": Array [
                                                        Object {
                                                          "id": "SEL_1",
                                                          "name": "Real Madrid",
                                                          "price": 1.23,
                                                        },
                                                        Object {
                                                          "id": "SEL_2",
                                                          "name": "Barcelona",
                                                          "price": 2.05,
                                                        },
                                                      ],
                                                    },
                                                    Object {
                                                      "id": "MKT_2",
                                                      "name": "Player to Score First",
                                                      "selections": Array [
                                                        Object {
                                                          "id": "SEL_3",
                                                          "name": "Ronaldo",
                                                          "price": 1.15,
                                                        },
                                                        Object {
                                                          "id": "SEL_4",
                                                          "name": "Messi",
                                                          "price": 1.3,
                                                        },
                                                        Object {
                                                          "id": "SEL_5",
                                                          "name": "Bale",
                                                          "price": 1.35,
                                                        },
                                                      ],
                                                    },
                                                  ],
                                                  "name": "Real Madrid vs Barcelona",
                                                }
                              }
                    />
                    <Event
                              event={
                                        Object {
                                                  "id": "EVT_2",
                                                  "markets": Array [
                                                    Object {
                                                      "id": "MKT_3",
                                                      "name": "Team to Win",
                                                      "selections": Array [
                                                        Object {
                                                          "id": "SEL_6",
                                                          "name": "Atletico",
                                                          "price": 1.4,
                                                        },
                                                        Object {
                                                          "id": "SEL_7",
                                                          "name": "Malaga",
                                                          "price": 3.05,
                                                        },
                                                      ],
                                                    },
                                                  ],
                                                  "name": "Atletico Madrid vs Malaga",
                                                }
                              }
                    />
                    
          </div>
</div>,
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": <div
            className="inner"
>
            <NavBar />
            <BetSlip />
            <Event
                        event={
                                    Object {
                                                "id": "EVT_1",
                                                "markets": Array [
                                                  Object {
                                                    "id": "MKT_1",
                                                    "name": "Team to Win",
                                                    "selections": Array [
                                                      Object {
                                                        "id": "SEL_1",
                                                        "name": "Real Madrid",
                                                        "price": 1.23,
                                                      },
                                                      Object {
                                                        "id": "SEL_2",
                                                        "name": "Barcelona",
                                                        "price": 2.05,
                                                      },
                                                    ],
                                                  },
                                                  Object {
                                                    "id": "MKT_2",
                                                    "name": "Player to Score First",
                                                    "selections": Array [
                                                      Object {
                                                        "id": "SEL_3",
                                                        "name": "Ronaldo",
                                                        "price": 1.15,
                                                      },
                                                      Object {
                                                        "id": "SEL_4",
                                                        "name": "Messi",
                                                        "price": 1.3,
                                                      },
                                                      Object {
                                                        "id": "SEL_5",
                                                        "name": "Bale",
                                                        "price": 1.35,
                                                      },
                                                    ],
                                                  },
                                                ],
                                                "name": "Real Madrid vs Barcelona",
                                              }
                        }
            />
            <Event
                        event={
                                    Object {
                                                "id": "EVT_2",
                                                "markets": Array [
                                                  Object {
                                                    "id": "MKT_3",
                                                    "name": "Team to Win",
                                                    "selections": Array [
                                                      Object {
                                                        "id": "SEL_6",
                                                        "name": "Atletico",
                                                        "price": 1.4,
                                                      },
                                                      Object {
                                                        "id": "SEL_7",
                                                        "name": "Malaga",
                                                        "price": 3.05,
                                                      },
                                                    ],
                                                  },
                                                ],
                                                "name": "Atletico Madrid vs Malaga",
                                              }
                        }
            />
            
</div>,
          "className": "bets",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": Array [
              <NavBar />,
              <BetSlip />,
              Array [
                <Event
                  event={
                                    Object {
                                                      "id": "EVT_1",
                                                      "markets": Array [
                                                        Object {
                                                          "id": "MKT_1",
                                                          "name": "Team to Win",
                                                          "selections": Array [
                                                            Object {
                                                              "id": "SEL_1",
                                                              "name": "Real Madrid",
                                                              "price": 1.23,
                                                            },
                                                            Object {
                                                              "id": "SEL_2",
                                                              "name": "Barcelona",
                                                              "price": 2.05,
                                                            },
                                                          ],
                                                        },
                                                        Object {
                                                          "id": "MKT_2",
                                                          "name": "Player to Score First",
                                                          "selections": Array [
                                                            Object {
                                                              "id": "SEL_3",
                                                              "name": "Ronaldo",
                                                              "price": 1.15,
                                                            },
                                                            Object {
                                                              "id": "SEL_4",
                                                              "name": "Messi",
                                                              "price": 1.3,
                                                            },
                                                            Object {
                                                              "id": "SEL_5",
                                                              "name": "Bale",
                                                              "price": 1.35,
                                                            },
                                                          ],
                                                        },
                                                      ],
                                                      "name": "Real Madrid vs Barcelona",
                                                    }
                  }
/>,
                <Event
                  event={
                                    Object {
                                                      "id": "EVT_2",
                                                      "markets": Array [
                                                        Object {
                                                          "id": "MKT_3",
                                                          "name": "Team to Win",
                                                          "selections": Array [
                                                            Object {
                                                              "id": "SEL_6",
                                                              "name": "Atletico",
                                                              "price": 1.4,
                                                            },
                                                            Object {
                                                              "id": "SEL_7",
                                                              "name": "Malaga",
                                                              "price": 3.05,
                                                            },
                                                          ],
                                                        },
                                                      ],
                                                      "name": "Atletico Madrid vs Malaga",
                                                    }
                  }
/>,
                "",
              ],
            ],
            "className": "inner",
          },
          "ref": null,
          "rendered": Array [
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": undefined,
              "nodeType": "class",
              "props": Object {},
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": "EVT_1",
              "nodeType": "class",
              "props": Object {
                "event": Object {
                  "id": "EVT_1",
                  "markets": Array [
                    Object {
                      "id": "MKT_1",
                      "name": "Team to Win",
                      "selections": Array [
                        Object {
                          "id": "SEL_1",
                          "name": "Real Madrid",
                          "price": 1.23,
                        },
                        Object {
                          "id": "SEL_2",
                          "name": "Barcelona",
                          "price": 2.05,
                        },
                      ],
                    },
                    Object {
                      "id": "MKT_2",
                      "name": "Player to Score First",
                      "selections": Array [
                        Object {
                          "id": "SEL_3",
                          "name": "Ronaldo",
                          "price": 1.15,
                        },
                        Object {
                          "id": "SEL_4",
                          "name": "Messi",
                          "price": 1.3,
                        },
                        Object {
                          "id": "SEL_5",
                          "name": "Bale",
                          "price": 1.35,
                        },
                      ],
                    },
                  ],
                  "name": "Real Madrid vs Barcelona",
                },
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            Object {
              "instance": null,
              "key": "EVT_2",
              "nodeType": "class",
              "props": Object {
                "event": Object {
                  "id": "EVT_2",
                  "markets": Array [
                    Object {
                      "id": "MKT_3",
                      "name": "Team to Win",
                      "selections": Array [
                        Object {
                          "id": "SEL_6",
                          "name": "Atletico",
                          "price": 1.4,
                        },
                        Object {
                          "id": "SEL_7",
                          "name": "Malaga",
                          "price": 3.05,
                        },
                      ],
                    },
                  ],
                  "name": "Atletico Madrid vs Malaga",
                },
              },
              "ref": null,
              "rendered": null,
              "type": [Function],
            },
            "",
          ],
          "type": "div",
        },
        "type": "div",
      },
      "type": [Function],
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
